loops_lab.js

```diff
@@ -0,0 +1,203 @@
+// <!-- # Loops Lab
```

Comment:
I think you understand the concept and usage of for and while loops but still needs more practice when it comes to setting logics inside if statement. It takes practice but I believe you will get there. Keep it up!